docs/pages/installation.mdx

```diff
@@ -235,3 +240,10 @@ any OS supported by the [Golang toolchain](https://github.com/golang/go/wiki/Min
 
 \[2] *Teleport server does not run on Windows yet, but `tsh` (the Teleport client)
 supports most features on Windows 10 and later.*
+
+## Next steps
```

Comment:
nit: the rest of the headings on this page have an empty line after the header and before the content begins. I find this more readable.